### ISSUE TITLE
Discord link

### ISSUE THREAD

**Issue by drevilz4l** [foone/3DMMForever#63]
Did something happen to the discord for this project? I don't seem to be on it anymore.

**Comment by jayrod246** [comment]
Yes, the discord server formerly known as Open3DMM was deleted. It was a decision I made for various reasons, mostly personal ones. In retrospect it may have been rash, but I don’t regret it either.
